CONTRIBUTING.md

```diff
@@ -271,3 +271,5 @@ https://jupyterlab-tutorial.readthedocs.io/en/latest/index.html.
 
 - The npm modules are fully compatible with Node/Babel/ES6/ES5. Simply
 omit the type declarations when using a language other than TypeScript.
+
+- For more information, read the [documentation](http://jupyterlab-tutorial.readthedocs.io/en/latest/).
```

Comment:
Thanks!  Our documentation has moved to `http://jupyterlab.readthedocs.io/en/latest/`

Reply:
Also I was explicitly referring to http://jupyterlab.readthedocs.io/en/stable/developer/documentation.html

Reply:
Actually I meant http://jupyterlab.readthedocs.io/en/stable/developer/documentation.html